modules/govuk/manifests/apps/publishing_api.pp

```diff
@@ -125,6 +128,7 @@
   $draft_content_store_bearer_token = undef,
   $router_api_bearer_token = undef,
   $suppress_draft_store_502_error = '',
+  $unicorn_worker_processes = undef,
```

Comment:
Perhaps set this to `2` explicitly so everyone knows what the default value is without having to 'know' or Google it?

Reply:
👍